basestar-schema/src/main/java/io/basestar/schema/from/FromSql.java

```diff
@@ -168,4 +169,26 @@ public String getReplacedSql(final Function<Schema<?>, String> replacer) {
 
         return FunctionSchema.getReplacedDefinition(sql, using, replacer);
     }
+
+    public Pair<String, List<Object>> getReplacedSqlWithBindings(final Function<Schema<?>, String> replacer, final List<Argument> arguments, final Map<String, Object> values) {
+
+        final String sql = getReplacedSql(replacer);
+
+        final List<Object> bindings = new ArrayList<>();
+        final StringBuffer str = new StringBuffer();
+
+        final Pattern pattern = Pattern.compile("\\$\\{(.*?)}");
+        final Matcher matcher = pattern.matcher(sql);
+        if (matcher.find()) {
+            final String name = matcher.group(1);
```

Comment:
is this replacement of the _first_ argument only? Should this be `while(matcher.find())`?